waspc/data/Generator/templates/react-app/src/auth/forms/Login.js

```diff
@@ -24,7 +24,7 @@ const LoginForm = () => {
   }
   
   return (
-    <form onSubmit={handleLogin}>
+    <form onSubmit={handleLogin} className="login-form auth-form">
```

Comment:
Added some class names to help with formating default login/signup forms.

Reply:
this is nice, finally making our signup forms prettier!